src/core/reactor.cc

```diff
@@ -4131,6 +4131,12 @@ class disk_config_params {
                         throw std::runtime_error(fmt::format("Configured number of queues {} is larger than the maximum {}",
                                                  _mountpoints.size(), _max_queues));
                     }
+
+                    d.read_bytes_rate = d.read_bytes_rate * d.rate_factor;
```

Comment:
```suggestion
                    d.read_bytes_rate *= d.rate_factor;
```

Reply:
also, if the only sources of `mountpoint_params` are 

- its default constructor if the mount pointer is not configured
- the yaml settings

can we instead have a tiny helper method exposed by `mountpoint_params` like:
```c++
struct mountpoint_params {
    uint64_t read_bytes_rate = std::numeric_limits<uint64_t>::max();
    // ...
    void apply_rate(float rate) {
        read_bytes_rate *= rate;
        write_bytes_rate *= rate;
        // ...
    }
};

namespace YAML {
template<>
struct convert<seastar::mountpoint_params> {
    static bool decode(const Node& node, seastar::mountpoint_params& mp) {
        // ...
        if (node["rate_factor"]) {
            mp.apply_rate(node["rate_factor"].as<float>());
        }
        /// btw, can even fail the conversion if `rate_factor` is 0.
        return true;
    }
};
```

Reply:
I think the first suggestion to use `*=` is better. Applied one